netdaemon3/config.json

```diff
@@ -26,6 +26,9 @@
     "app_config_folder": "/config/netdaemon3",
     "app_assembly": ""
   },
+  "services": [
```

Comment:
I am not sure I want to make this dependency for this. I consider creating entities still not that common.

Reply:
It will not be a dependency, it will only grant access if present.